### PR TITLE
Issue-#32-Doc-update Readme

### LINKED ISSUE #32: Update Readme

### PR DESCRIPTION
**Pull request by Sebberh**
Closes #32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +39,9 @@ Issue-#[issueNr]-[Title]
 Add "Closes #[issueNr]" to the description or link the issue manually in the GUI.
 
 ##### Statement of contributions:
-
+Communicating with Github API
+Remaking code skelleton with updated libs
+Documentation & tests
 ###### Sebastian Fagerlind
```
Comment (hengque) [inline README.md]:
```suggestion
###### Sebastian Fagerlind
Communicating with GitHub API
Remaking code skeleton with updated libs
Documentation & tests

```

**Review by hengque — CHANGES_REQUESTED** [review]
I think you might have meant to write the contribution under your name

File: README.md (hunk 1 above)
Comment (EleonoraBorzi) [reply README.md]: test

Comment (EleonoraBorzi) [reply README.md]: test

**Pull request merged**